test/it/client-listUsers.js

```diff
@@ -15,8 +20,18 @@ describe('client.listUsers()', () => {
   let _user;
 
   before(() => {
-    const newUser = utils.userWithPassword();
-    newUser.profile.nickName = faker.random.uuid();
+    const newUser = {
+      profile: {
+        firstName: 'John',
+        lastName: 'List-Users',
+        email: 'john-list-users@example.com',
+        login: 'john-list-users@example.com'
+      },
+      credentials: {
+        password: {value: 'Abcd1234'}
+      }
+    };
+    newUser.profile.nickName = 'johny-list-users';
```

Comment:
@robertjd - This is one of your original tests. Do you have a reason for updating the profile `nickname`? I could add a comment anyway.